src/Appwrite/Utopia/Request.php

```diff
@@ -27,6 +27,11 @@ public function __construct(SwooleRequest $request)
         parent::__construct($request);
     }
 
+    public function getSwoole(): SwooleRequest
```

Comment:
This is being used to manipulate the initial request to redirect to the REST API. Most of the uses could be moved into `set` methods on the request base class. 

There is one usage where I'm not sure how to avoid using this; when [creating a new request object](https://github.com/appwrite/appwrite/blob/575b882b306099309a25cac9ebb7a0f1568efc14/src/Appwrite/GraphQL/Resolvers.php#L299), as the constructor requires the swoole request. The new request is needed so the initial GraphQL request is not ended when the REST request completes.

Reply:
Managed to remove this by adding some `set` methods to the `Request` object (in utopia-php/framework), and adding a `clone` function in this class to avoid having to get the swoole request to create a new one.